### PR TITLE
Disable modifying files to be reviewed in form (bug 1197261)

### PR DESCRIPTION
**Pull request by wagnerand**

### CODE REVIEW

**Comment by wagnerand** [comment]
@kmaglione I don't understand why those tests are failing. What's wrong with the patch?

**Comment by kmaglione** [comment]
The tests are testing for behavior that you removed. You just need to remove those tests.

File: apps/editors/forms.py [hunk 1]
```diff
@@ -18,8 +18,7 @@
 from amo.urlresolvers import reverse
 from amo.utils import raise_required
 from applications.models import AppVersion
-from editors.helpers import (file_review_status, ReviewAddon, ReviewFiles,
-                             ReviewHelper)
+from editors.helpers import (file_review_status, ReviewAddon, ReviewHelper)
```
Comment (kmaglione) [inline apps/editors/forms.py]: Remove parens please.

File: apps/editors/forms.py [hunk 2]
```diff
@@ -331,8 +330,7 @@ def clean_addon_files(self):
 
 def get_review_form(data, request=None, addon=None, version=None):
     helper = ReviewHelper(request=request, addon=addon, version=version)
-    form = {ReviewAddon: ReviewAddonForm,
-            ReviewFiles: ReviewFileForm}[helper.handler.__class__]
+    form = {ReviewAddon: ReviewAddonForm}[helper.handler.__class__]
```
Comment (kmaglione) [inline apps/editors/forms.py]: Dict is no longer necessary. Just get rid of `form` and use `ReviewAddonForm` directly on the line below.

File: apps/editors/templates/editors/review.html [hunk 3]
```diff
@@ -163,7 +163,7 @@ <h3 id="history">
       </div>
 
       <div class="review-actions-section review-actions-files data-toggle"
-           data-value="{{ actions_minimal|join("|") }}"{% if allow_unchecking_files %} data-uncheckable="1"{% endif %}>
+           data-value="{{ actions_minimal|join("|") }}" data-uncheckable="1">
```
Comment (kmaglione) [inline apps/editors/templates/editors/review.html]: You may as well just get rid of the attribute and the JS code that relies on it.

File: apps/editors/tests/test_helpers.py [hunk 4]
```diff
@@ -229,11 +229,6 @@ def test_no_version(self):
                                       version=None)
         eq_(helper.review_type, 'pending')
 
-    def test_review_files(self):
-        for status in REVIEW_FILES_STATUSES:
```
Comment (kmaglione) [inline apps/editors/tests/test_helpers.py]: Need to get rid of this constant on line 32 and merge its values with the `REVIEW_ADDON_STATUSES` constant on the line above it. `test_nominated_review_time_set` and `test_preliminary_review_time_set` will also need to be merged, somehow.

**Comment by kmaglione** [comment]
r+wc

**Comment by wagnerand** [comment]
Done, all tests passing.

Thank you for the review!

**Comment by magopian** [comment]
Thanks a nice big chunk of deleted code, congratz!

r+

File: apps/editors/tests/test_helpers.py [hunk 5]
```diff
@@ -921,7 +916,8 @@ def test_nominated_review_time_set(self):
         for status in REVIEW_ADDON_STATUSES:
             for process in ['process_sandbox', 'process_preliminary',
                             'process_public']:
-                if (status == amo.STATUS_UNREVIEWED and
+                if ((status == amo.STATUS_UNREVIEWED or
+                        status == amo.STATUS_LITE) and
```
Comment (kmaglione) [inline apps/editors/tests/test_helpers.py]: I would've done `status in (amo.STATUS_UNREVIEWED, amo.STATUS_LITE)`, but this works too.

Comment (wagnerand) [inline apps/editors/tests/test_helpers.py]: I come from Java and JavaScript, I'm spoiled...

Comment (kmaglione) [inline apps/editors/tests/test_helpers.py]: Spoiled? In JavaScript that'd be `~[amo.STATUS_UNREVIEWED, amo.STATUS_LITE].indexOf(status)`

**Pull request merged**